web/ui-app-v2/src/screens/Feedback/components/ButtonToggle/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react";
+import { Card } from "../../../../components";
+import { Image } from "../../../../components";
+import "./index.css";
+import "../../../../styles/app.css";
+import ButtonToggle from "../../../../components/ButtonGroup";
```

Comment:
Instead of including components seperately, include via named imports.

`import { Card,Image, ButtonToggle } from "../../../../components";`